packages/shared/styles/components/SfBanner.scss

```diff
@@ -34,11 +34,20 @@ $banner--slim__title-font-size: $h1-font-size-desktop !default;
   background-repeat: no-repeat;
   background-position: bottom left;
   background-size: cover;
+  background: --background-color;
+  @if --m {
```

Comment:
just
```
background-color: var(--background-color);
background-image: var(--m);
```

Reply:
and please use a descriptive name like `--background-image-mobile` and `--background-image-desktop`